File: beginners/2-problemset-condicionais/1040-media3/media3.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double a, b, c, d, e, media, mediae;

    cin >> a >> b >> c >> d;

    media = ((a * 2) + (b * 3) + (c * 4) + d) / 10;

    if (media >= 7.0)
        cout << "Media: " << fixed << setprecision(1) << media << "\nAluno aprovado." << endl;
    else if (media < 5.0)
        cout << "Media: " << fixed << setprecision(1) << media << "\nAluno reprovado." << endl;
    else if (media >= 5.0 && media <= 6.9) {
        cin >> e;

        mediae = (e + media) / 2;

        cout << "Media: " << fixed << setprecision(1) << media << "\nAluno em exame." << endl;
        cout << "Nota do exame: " << fixed << setprecision(1) << e;

        if (mediae >= 5.0)
            cout << "\nAluno aprovado.\n" << "Media final: " << fixed << setprecision(1) << mediae << endl;
        else if (mediae < 5.0)
            cout << "Aluno reprovado.\n" << "Media final: " << fixed << setprecision(1) << mediae << endl;
    }

    return 0;
}

File: beginners/2-problemset-condicionais/Python/1040-media3/media3.py
n1, n2, n3, n4 = map(float, input().split())

media = ((n1 * 2) + (n2 * 3) + (n3 * 4) + (n4 * 1)) / 10

if media >= 7:
    print(f'Media: {media:.1f}\nAluno aprovado.')
elif media < 5:
    print(f'Media: {media:.1f}\nAluno reprovado.')
else:
    print(f'Media: {media:.1f}\nAluno em exame.')
    ne = float(input())

    print(f'Nota do exame: {ne:.1f}')
    mediae = (media + ne) / 2

    if (mediae >= 5):
        print(f'Aluno aprovado.\nMedia final: {mediae:.1f}')
    else:
        print(f'Aluno reprovado.\nMedia final: {mediae:.1f}')
